src/Concerns/InteractsWithCommands.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+
+/** @noinspection PhpUnhandledExceptionInspection */
+
+namespace DefStudio\Telegraph\Concerns;
+
+use DefStudio\Telegraph\Telegraph;
+
+/**
+ * @mixin Telegraph
+ */
+
+trait InteractsWithCommands
+{
+    public function getMyCommands(): Telegraph
```

Comment:
This should be better `getRegisteredCommands()` for consistency with `registerBotCommands` and unregisterBotCommands` methods. Could you update your code to reflect this?

also, the same method (`getRegisteredCommands`) and its tests, should be added in `TelegraphBot` model